woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -162,6 +162,21 @@ def test_init_accessor_with_schema_errors(sample_df):
         iloc_df.ww.init(schema=schema)
 
 
+def test_accessor_with_schema_parameter_warning(sample_df):
+    xfail_dask_and_koalas(sample_df)
+
+    schema_df = sample_df.copy()
+    schema_df.ww.init(name='test_schema', semantic_tags={'id': 'test_tag'}, index='id')
+    schema = schema_df.ww.schema
+
+    head_df = schema_df.head(2)
+    assert head_df.ww.schema is None
+
+    warning = "A schema was provided and the following parameters were ignored: index, semantic_tags"
+    with pytest.warns(Warning, match=warning):
+        head_df.ww.init(index='id', semantic_tags={'id': 'test_tag'}, schema=schema)
```

Comment:
Let's use some different values for index and semantic tags, just so we can confirm below that the values from the schema are being used and not the values we are passing for these parameters.

Reply:
That's a good call. There are issues with _validate_accessor_params if we use different values since it doesn't know they are being ignored. I'll get to this later today.

Reply:
Just kidding they were all quick fixes. Let me know if there's anything else!